R/theme_turnPlot.R

```diff
@@ -13,9 +13,9 @@ theme_turnPlot <- function(base_size = 11, base_family = "serif", ticks = TRUE)
     ticks = ticks
   ) %+replace%
     theme(
-      legend.position = "none",
+#      legend.position = "none",
```

Comment:
```suggestion
```
best to remove commented out code altogether when you're out of the troubleshooting phase